ops/ansible/roles/bfd-server/tasks/install.yml

```diff
@@ -1,14 +1,5 @@
 ---
 
-- name: Install Pre-requisites
-  yum:
-    name: "{{ item }}"
-    state: present
-  with_items:
-    # Needed to supply `keytool` and also to run the app server.
-    - java-1.8.0-openjdk-devel
-  become: true
-
```

Comment:
Java is already installed during the platinum build, not required here.

Reply:
FYI: Undoing this as part of the capsule changes. Two reasons:

1. We have tests that also use the role.
2. It's not listed as a pre-req for the role, so the role should provide it.